Index: FFmpeg/libavutil/common.h
===================================================================
--- FFmpeg/libavutil/common.h
+++ FFmpeg/libavutil/common.h
@@ -28,7 +28,9 @@
 
 #include <ctype.h>
 #include <errno.h>
+#ifndef FB_WIN
 #include <inttypes.h>
+#endif
 #include <limits.h>
 #include <math.h>
 #include <stdio.h>
Index: libavformat/tcp.c
===================================================================
--- FFmpeg/libavformat/tcp.c
+++ FFmpeg/libavformat/tcp.c
@@ -179,7 +179,12 @@
         if (ret < 0)
             return ret;
     }
-    ret = send(s->fd, buf, size, 0);
+//    ret = send(s->fd, buf, size, 0);
+    #ifdef __APPLE__
+      ret = send(s->fd, buf, size, SO_NOSIGPIPE);
+    #else
+      ret = send(s->fd, buf, size, MSG_NOSIGNAL);
+    #endif
     return ret < 0 ? ff_neterrno() : ret;
 }
 
Index: FFmpeg/libavformat/rtpdec_mpeg4.c
===================================================================
--- FFmpeg/libavformat/rtpdec_mpeg4.c
+++ FFmpeg/libavformat/rtpdec_mpeg4.c
@@ -203,6 +203,7 @@
     }
 
     if (codec->codec_id == AV_CODEC_ID_AAC) {
+        while((*attr==' ' || *attr==';')) attr++;
         /* Looking for a known attribute */
         for (i = 0; attr_names[i].str; ++i) {
             if (!av_strcasecmp(attr, attr_names[i].str)) {

